layouts/layouts/src/main/java/com/microsoft/device/dualscreen/layouts/FoldableLayout.kt

```diff
@@ -203,7 +203,7 @@ open class FoldableLayout @JvmOverloads constructor(
         if (layoutController.isChangingContent) {
             addContentChangedListener(
                 object : ContentChangedListener {
-                    override fun contentChanged() {
+                    override fun contentChanged(screenMode: ScreenMode?) {
```

Comment:
I think I may be missing something in the code, but how is this new parameter used?

Reply:
You need this parameter inside Activity to know what childs you should bind, because you can have different childs depending on the single-screen/dual-screen mode. Otherwise, you should use your own WM callback inside activity and sync with the callback from the FoldableLayout, and the result will be some ugly code.